editoast/editoast_models/src/rolling_stock.rs

```diff
@@ -1,3 +1,3 @@
 mod train_category;
-pub use train_category::TrainCategories;
-pub use train_category::TrainCategory;
+pub use train_category::TrainMainCategories;
```

Comment:
The module may need a rename too?

Reply:
Not really, because there will be an enum called `TrainCategory`, with all other types of categories, `TrainMainCategory` and `SubCategory`.

Reply:
Or maybe it better to separate into different modules 👍